src/Common/Misc/ConvertProxyProperty/index.js

```diff
@@ -92,14 +92,15 @@ export function proxyPropToProp(property, ui) {
   const depStatus = depList ? Boolean(Number(depList.pop())) : true;
   const depValue = depList ? depList.pop() : null;
   const depId = depList ? depList.join(':') : null;
-  const searchString = [ui.name].concat(property.value).join(' ').toLowerCase();
+  const searchString = [ui.name].concat(property.value, depList || []).join(' ').toLowerCase();
```

Comment:
In ParaView, if you search for "loc" on the contour filter, you see the 'Point Locator' group of properties, including its dependent properties. So I added the items from the depList into the searchString to make that happen